experimental/segmenter/src/provider.rs

```diff
@@ -81,10 +84,20 @@ pub struct UCharDictionaryBreakDataV1<'data> {
     pub trie_data: ZeroVec<'data, u16>,
 }
 
+#[cfg(not(feature = "lstm"))]
 pub const ALL_KEYS: [ResourceKey; 5] = [
     LineBreakDataV1Marker::KEY,
     GraphemeClusterBreakDataV1Marker::KEY,
     WordBreakDataV1Marker::KEY,
     SentenceBreakDataV1Marker::KEY,
     UCharDictionaryBreakDataV1Marker::KEY,
 ];
+#[cfg(feature = "lstm")]
+pub const ALL_KEYS: [ResourceKey; 6] = [
+    LineBreakDataV1Marker::KEY,
+    GraphemeClusterBreakDataV1Marker::KEY,
+    WordBreakDataV1Marker::KEY,
+    SentenceBreakDataV1Marker::KEY,
+    UCharDictionaryBreakDataV1Marker::KEY,
+    LstmDataV1Marker::KEY,
```

Comment:
You should be able to `cfg` the entry instead of repeating the whole list

Reply:
This array is fixed size, so I guess this change causes build error without lstm. So I will add another condition for fixed size.